### PR TITLE
New package: vpsm-0.1

### PR DESCRIPTION
**Pull request by sinetoami**

### CODE REVIEW

File: srcpkgs/vpsm/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+depends="bash xbps xtools git ripgrep hub"
+short_desc="Void-Packages Sources Management wrapper for XBPS-SRC"
+maintainer="Sinésio Neto <sinetoami@gmail.com>"
+license="MIT"
+homepage="https://github.com/sinetoami/vpsm"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=be62fdb06b5ce6d8555f375f8b8d2e8c78e2a2dea191f1809345601e5e933ecd
+
+post_install() {
+cd ${wrksrc}
```
Comment (maxice8) [inline srcpkgs/vpsm/template]: please use proper indentation inside functions, 1 tab

File: srcpkgs/vpsm/INSTALL.msg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+For appropriate use, export the XBPS_DISTDIR env var
+# .zshrc
+echo "export XBPS_DISTDIR=$HOME/.void-packages" >> ~/.zshrc
+
```
Comment (maxice8) [inline srcpkgs/vpsm/INSTALL.msg]: Why not just modify your script to use `xdistdir` from xtools ?

Comment (sinetoami) [reply srcpkgs/vpsm/INSTALL.msg]: It's used. But if the `XBPS_DISTDIR` is not the `xdistdir` return a message asking to export it. That's what happened in my tests.

**Comment by maxice8** [comment]
On Wed, Oct 24, 2018 at 09:54:52PM +0000, Sinésio Neto wrote:
> sinetoami commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,6 @@
> +For appropriate use, export the XBPS_DISTDIR env var
> +# .zshrc
> +echo "export XBPS_DISTDIR=$HOME/.void-packages" >> ~/.zshrc
> +
> 
> It's used. But if the `XBPS_DISTDIR` is not the `xdistdir` return a message asking to export it. That's what happened in my tests.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/4142#discussion_r227973925

xdistdir tries to find in the most common places and errors out if it doesn't.

So just use xdistdir and if it doesn't work then error out to the user.

**Comment by sinetoami** [comment]
> On Wed, Oct 24, 2018 at 09:54:52PM +0000, Sinésio Neto wrote: sinetoami commented on this pull request. > @@ -0,0 +1,6 @@ +For appropriate use, export the XBPS_DISTDIR env var +# .zshrc +echo "export XBPS_DISTDIR=$HOME/.void-packages" >> ~/.zshrc + It's used. But if the `XBPS_DISTDIR` is not the `xdistdir` return a message asking to export it. That's what happened in my tests. -- You are receiving this because you commented. Reply to this email directly or view it on GitHub: [#4142 (comment)](https://github.com/void-linux/void-packages/pull/4142#discussion_r227973925)
> xdistdir tries to find in the most common places and errors out if it doesn't. So just use xdistdir and if it doesn't work then error out to the user.

ok  :+1:

File: srcpkgs/vpsm/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+depends="bash xbps xtools git ripgrep hub"
+short_desc="Void-Packages Sources Management wrapper for XBPS-SRC"
+maintainer="Sinésio Neto <sinetoami@gmail.com>"
+license="MIT"
+homepage="https://github.com/sinetoami/vpsm"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=3e7e902aea8245ba8a30e1a1d522dce4c98c6027672a9e0493e39d31025a2473
+
+post_install() {
+	cd ${wrksrc}
```
[the remark this answers is not in the document]
Comment (sinetoami) [reply srcpkgs/vpsm/template]: I've been looking for the answer for a while. The `Makefile` has the following instructions:
```
PHONY: install
install:
	cp vpsm $(PREFIX)/bin/vpsm
	cp man/vpsm.1 $(PREFIX)/share/man/man1/vpsm.1
```
 But for some reason (that I did not find out) the `vpsm` isn't put in the right place after install. Actually it looks like it's not copied.

[the remark this answers is not in the document]
Comment (sinetoami) [reply srcpkgs/vpsm/template]: Ok. Thanks for helping me.

[the remark this answers is not in the document]
Comment (sinetoami) [reply srcpkgs/vpsm/template]: Done. I guess found the solution.

**Comment by CameronNemo** [comment]
Please squash your commits.

**Comment by sinetoami** [comment]
> Please squash your commits.

Did I set the squash correctly?

**Comment by CameronNemo** [comment]
Yep looks good.

File: srcpkgs/vpsm/template [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'vpsm'
+pkgname=vpsm
+version=0.1.2
+revision=1
+build_style=gnu-makefile
+depends="bash xbps xtools ripgrep hub"
```
Comment (maxice8) [inline srcpkgs/vpsm/template]: no need to depend on xbps, it is always installed.

**Pull request merged**